scripts/src/public/services/orders.ts

```diff
@@ -188,6 +198,8 @@ async function createP2PExternalOrder(sender: User, jwt: ExternalPayToUserOrderJ
 async function createNormalEarnExternalOrder(recipient: User, jwt: ExternalEarnOrderJWT) {
 	const app = await Application.findOneById(recipient.appId);
 
+	throwOnUserEarnLimit(recipient.id, "total_earn", app!.config.limits.hourly_user_earn, moment.duration({ hours: 1 }), jwt.offer.amount);
```

Comment:
1. title should be "user_earn".
2. I think that as a util function, `throwOnUserEarnLimit`, should put this string by itself instead of receiving it as an input. - but lets schedule this to a different PR